news/2022-09-01-how-zetera-became-a-medal-hunter.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "How Zetera became a Medal Hunter"
+date: 2022-09-31 00:00:00 +0000
```

Comment:
ah, yes, september 31

Reply:
just a placeholder lol didnt even realize what I wrote, I will change it later